guides/common/modules/con_planning-organization-and-location-context.adoc

```diff
@@ -0,0 +1,27 @@
+[id="planning-organization-and-location-context_{context}"]
+= Planning organization and location context
+
+Context in {Project} consists of organizations and locations.
+You can associate most resources (such as hosts, subnets, or domains) with at least one organization and location context.
+
+Resources and users can generally only access resources within their own context, which makes configuring organizations and locations an integral part of access management in {Project}.
+
+[IMPORTANT]
+====
+If you use host groups to bundle provisioning and configuration information, avoid mismatching resources from mutually exclusive contexts.
+For example, setting a subnet from one organization or location and a compute resource from a different organization or location creates an invalid host group.
+====
+
+Some resources in {Project}, such as Ansible roles and operating systems, are not part of any organization or location context.
```

Comment:
Should this also mention that most of the things comming from katello, are strictly scoped to exactly one organization? Or is this strictly for foreman and plugins without katello?

Reply:
This whole [guide is Katello-specific](https://github.com/theforeman/foreman-documentation/blob/e5b4f53cfe15ca3b434b65fa97e76c29eb26f351/guides/doc-Planning_for_Project/master.adoc?plain=1#L7). The only other place where this module appears is in [Managing organizations and locations](https://docs.theforeman.org/nightly/Managing_Organizations_and_Locations/index-katello.html#Introduction_to_Organization_and_Location_Context_managing-organizations-locations), which is also not published in any non-Katello context but if I'm reading [this right](https://github.com/theforeman/foreman-documentation/blob/master/guides/doc-Managing_Organizations_and_Locations/master.adoc?plain=1), it could be in the future.

I think it would help if I hide this from non-Katello/non-Satellite builds. Not sure about orcharhino. Any opinions @maximiliankolb?

Reply:
orcharhino always includes Katello. -> I propose to make it Foreman-only; see my suggestion above.